package.json

```diff
@@ -12,7 +12,7 @@
     "test": "mocha test/ServerSpec.js & mocha test/setup.js test/ClientSpec.js",
     "dev": "webpack -d --watch",
     "build": "webpack -p",
-    "start": "node server/server.js",
+    "start": "nodemon server/server.js",
```

Comment:
I think this will just need me to add in a Procfile for heroku later, no worries. Everything is looking pretty great!